src/member/member.ts

```diff
@@ -22,14 +22,13 @@ export type MemberStorageItem = {
 };
 
 export type Pagination = {
-  totalItems: number;
-  totalPages: number;
-  currentPage: number;
+  totalCounts: number;
+  page: number;
   pageSize: number;
 };
 
 export type MemberStorageItemsResponse = {
-  items: MemberStorageItem[];
+  data: MemberStorageItem[];
   pagination: Pagination;
 };
```

Comment:
Since we have the generic type `Paginated` https://github.com/graasp/graasp-query-client/blob/94f7d69b370a79a934ed207960c85e301fa72797/src/types.ts#L72 in query-client I don't think we need to define this type. 

My idea is that we can use `Paginated<MemberStorageItem>` and this will be very clear for anyone using that type. Also having a single type that defines pagination allows us to update what we return for paginated items once instead of having multiple different implementation of the pagination mechanism.